frontend/lib/src/components/elements/ArrowVegaLiteChart/styled-components.ts

```diff
@@ -14,7 +14,82 @@
  * limitations under the License.
  */
 
+import { CSSObject } from "@emotion/react"
 import styled from "@emotion/styled"
+import { transparentize } from "color2k"
+
+import { EmotionTheme } from "@streamlit/lib/src/theme"
+
+export const StyledVegaLiteChartTooltips = (
+  theme: EmotionTheme
+): CSSObject => ({
+  "#vg-tooltip-element": {
+    visibility: "hidden",
+    position: "fixed",
+    fontFamily: theme.genericFonts.bodyFont,
+    color: theme.colors.bodyText,
+    border: `${theme.sizes.borderWidth} solid ${theme.colors.borderColor}`,
+    backgroundColor: transparentize(theme.colors.bgColor, 0.05),
+    fontSize: theme.fontSizes.twoSm,
+    boxShadow: "rgb(0 0 0 / 16%) 0px 1px 4px",
+    maxWidth: theme.sizes.maxChartTooltipWidth,
+    padding: `${theme.spacing.xs} ${theme.spacing.md}`,
+    borderRadius: theme.radii.default,
+    zIndex: theme.zIndices.vegaTooltips,
+
+    "&.visible": {
+      visibility: "visible",
+    },
+
+    h2: {
+      marginTop: theme.spacing.none,
+      marginBottom: theme.spacing.sm,
+      fontSize: theme.fontSizes.sm,
+    },
+
+    td: {
+      border: "none",
+    },
+
+    table: {
+      borderSpacing: 0,
+
+      tr: {
+        border: "none",
+
+        td: {
+          overflow: "hidden",
+          textOverflow: "ellipsis",
+          paddingTop: theme.spacing.threeXS,
+          paddingBottom: theme.spacing.threeXS,
+        },
+
+        "td.key": {
+          // This should use a max of 40% of the available the width (- padding):
```

Comment:
nit: // This should use a max of 40% of the available width
(one 'the' too much in front of width)